packages/react-components/react-shared-contexts/src/index.ts

```diff
@@ -34,5 +34,17 @@ export type { BackgroundAppearanceContextValue } from './BackgroundAppearanceCon
 
 export { PortalMountNodeProvider, usePortalMountNode } from './PortalMountNodeContext';
 
-export { AnnounceProvider as AnnounceProvider_unstable, useAnnounce as useAnnounce_unstable } from './AnnounceContext';
-export type { AnnounceContextValue as AnnounceContextValue_unstable } from './AnnounceContext';
+export {
+  AnnounceProvider,
+  /** @deprecated Use AnnounceProvider instead. */
```

Comment:
Im wondering how this aligns with our overal API approach ? AFAIR all hooks etc should remain under `_unstable` suffix pattern. Did this change in recent past ? Do we have docs/reasoning for this ? ty!

Reply:
> Im wondering how this aligns with our overal API approach ?

_Overall, I don't see a reason to introduce more unstable APIs as we can't break them._

These contexts should and the hook should be merged without `_unstable` suffix in the first place. As they are intended to be used publicly. The suffix also creates an impression that these APIs are not ready to be used, while they are 🙃

> AFAIR all hooks etc should remain under _unstable suffix pattern. Did this change in recent past ? Do we have docs/reasoning for this ? ty!

The agreement was about hooks that are component bits i.e. `useImage()`, `useTree()`, etc. and we keep this convention. However, all other hooks are just bits API as other things that's why we have `useEventCallback()`, `useFocusableGroup()`.

Overall, with `-preview` packages we have a clearer & better indicator of API stability. Does it make sense?

Reply:
yup ty!